frontend/src/components/accounts/AccountRow.tsx

```diff
@@ -0,0 +1,110 @@
+import moment from "moment";
+import React from "react";
+import { Row, Col } from "react-bootstrap";
+import Link from "next/link";
+
+import AccountsApi from "../../libraries/explorer-wamp/accounts";
+import Balance from "../utils/Balance";
+
+export interface Props {
+  accountId: string;
+  timestamp: any;
+}
+
+export interface State {
+  amount: string;
+}
+
+export default class extends React.Component<Props, State> {
+  state: State = {
+    amount: ""
+  };
+
+  _getDetail = async () => {
+    const detail = await new AccountsApi().getAccountInfo(this.props.accountId);
+    this.setState({ amount: detail.amount });
```

Comment:
Given you only need the amount, use `AccountsApi().queryAccount(this.props.accountId)` instead of `getAccountInfo`.